src/store/modules/app.store.js

```diff
@@ -19,8 +19,9 @@ export default {
         setAppIsReady: ({ commit }, { dispatcher }) => {
             commit('setAppIsReady', { dispatcher })
         },
-        mapModuleReady: ({ commit }, { dispatcher }) => {
+        mapModuleReady: ({ commit, dispatch }, { dispatcher }) => {
             commit('mapModuleReady', { dispatcher })
+            dispatch('clearLoadingBar4MapLoading', { dispatcher })
```

Comment:
as you are dispatching something that doesn't belong to this store "module", the best course of action would be to do that in a store plugin.
Maybe the one that already manages the app readiness would be a good candidate?